src/components/constants.js

```diff
@@ -1,3 +1,3 @@
-export const hothStart = new Date('2021-02-05T20:00:00-08:00');
-export const hothEnd = new Date('2021-02-07T21:00:00-08:00');
-export const applyDeadline = new Date('2021-02-01T23:59:59-08:00');
+export const hothStart = new Date('2022-02-26T20:00:00-08:00');
+export const hothEnd = new Date('2022-02-28T21:00:00-08:00');
```

Comment:
HOTH 9 will only be one day so set the `hothEnd` date to be the same day as `hothStart` + 12 hours (it's a 12 hour hackathon). You won't need to change any other code since in `Banner.js` if the start and end date are the same. The banner will display a singular date instead of a range.